bom/runtime/pom.xml

```diff
@@ -588,11 +606,13 @@
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-test-h2</artifactId>
                 <version>${project.version}</version>
+                <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-test-kubernetes-client</artifactId>
                 <version>${project.version}</version>
+                <scope>test</scope>
```

Comment:
We decided against adding the `scope` in the BOM and we explicitly removed them. It's considered bad practice.

Reply:
Ok, removed scope test.